src/routes/Transactions/routes/Asset/TransactionsAssetView.js

```diff
@@ -123,7 +123,7 @@ function TransactionsAssetView({
             </div>
             <div className='send' title={t`Send asset`}>
               <JFlatButton
-                to={`/digital-assets/send?asset=${asset.blockchainParams.address}`}
+                to={`/send?asset=${asset.blockchainParams.address}`}
```

Comment:
Do we have check before, that `asset.blockchainParams.address` exists?

Reply:
I guess, we need to think about more stable (without "maybe" fields) and highly specialized record structures, in future

Reply:
Users can change the link and set the `asset` value to `blahblah`, so the check should be definitely placed on target page, not here.